packages/blade/src/components/Box/Box.tsx

```diff
@@ -150,6 +152,22 @@ const Box = (props: BoxProps): JSX.Element => {
     validateBackgroundProp(props.backgroundColor);
   }, [props.backgroundColor]);
 
+  React.useEffect(() => {
```

Comment:
how till this work on SSR 🤔 

can you add this on SSR test?

Reply:
Oh it won't. It will work during hydration part of SSR rather than renderToString part. Should be fine right? it will still fail in dev-server.

Reply:
added test for errors in web, ssr, and native